gatsby-node.js

```diff
@@ -305,8 +306,11 @@ const localFileCheck = async (
       // the cleanKey here
       data[key].forEach((attachment) => {
         const ext = mime.getExtension(attachment.type); // unknown type returns null
+        // get the filename from the airtable metadata instead of the remote file
+        const airtableFileName = path.parse(attachment.filename).name;
```

Comment:
```suggestion
        // get the filename from the airtable metadata instead of the remote file
        const airtableFile = path.parse(attachment.filename);
        // console.log(airtableFile);
        let ext = airtableFile.ext;
        if (!ext) {
          const deriveExt = mime.getExtension(attachment.type); // unknown type returns null
          ext = !!deriveExt ? `.${deriveExt}` : undefined;
        }
```

I think considering we are using `path.parse()` in a greater way, we should use the file extension there and only attempt to derive if it doesn't have one.